tests/test_framework/revault_network.py

```diff
@@ -30,58 +30,96 @@ def __init__(
         self.postgres_pass = postgres_pass
         self.postgres_host = postgres_host
         self.coordinator_port = reserve()
-        self.cosigners_ports = []
 
         self.stk_wallets = []
+        self.stkman_wallets = []
         self.man_wallets = []
 
         self.csv = None
 
-    def deploy(self, n_stakeholders, n_managers, csv=None):
+    def deploy(self, n_stakeholders, n_managers, n_stkmanagers=0, csv=None):
         """
         Deploy a revault setup with {n_stakeholders} stakeholders, {n_managers}
-        managers and optionally fund it with {funding} sats.
+        managers.
```

Comment:
`, and {n_stkmanagers} stakeholders_managers` ?

Reply:
meh. It's self explanatory, i should remove it entirely if i retouch